pulp_python/app/tasks/sync.py

```diff
@@ -90,8 +91,12 @@ async def run(self):
         """
         If includes is specified, then only sync those,else try to sync all other packages
         """
+        if self.remote.proxy_url:
+            environ['http_proxy'] = self.remote.proxy_url
```

Comment:
Let's add a TODO to refactor Bandersnatch so that the internal API objects take configuration parameters rather than grabbing them straight from the environment.  Maybe as a comment here + an issue in their tracker.

This is fine for now, though. Not a super high priority to change.